nss-day-cohort-47/reactive-nutshell-the-dial-up-dudes
# Direct Message

# Description
Added ability to edit an @user message so that it changes to other @user. 
Fixes # (issue)
## Type of change
- [x ] New feature (non-breaking change which adds functionality)
# Testing Instructions
Send a message @ a user. It should have a receiverID for the user you sent the message to. Now edit it to @ a different user. The receiverID should have updated. 
# Checklist:
- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x I have commented my code, particularly in hard-to-understand areas
- [ x] My changes generate no new warnings or errors
- [ x] I have added test instructions that prove my fix is effective or that my feature works
